src/app/app.component.ts

```diff
@@ -66,16 +80,30 @@ export class AppComponent implements OnInit, OnDestroy {
     - Subscribe to changes
     - Check the received value using the areAllValuesTrue function and pass them to the isLoading variable. */
     // YOUR CODE STARTS HERE
+
+    combineLatest([
+      this.mockDataService.getPlanetLoader(),
+      this.mockDataService.getCharactersLoader(),
+    ]).
+    pipe(takeUntil(this.destroy$)).
+    subscribe(
```

Comment:
One small thing, you also can do like this, if you want
```
subscribe(
      (loaderStates: boolean[]) =>
        (this.isLoading = this.isSomeValuesTrue(loaderStates))
    )
```

Reply:
it is a good idea